core/src/main/scala/org/apache/spark/api/r/SerDe.scala

```diff
@@ -166,6 +166,24 @@ private[spark] object SerDe {
     }
   }
 
+  def readEnv(in: DataInputStream): java.util.Map[Object, Object] = {
```

Comment:
Minor comment - R env's are restricted to have String keys, so readMap is still a good name as this is probably a bit more general than R environments ?